frappe/utils/jinja.py

```diff
@@ -109,8 +109,7 @@ def get_jloader():
 
 		apps = frappe.get_hooks("template_apps")
 		if not apps:
-			apps = frappe.local.flags.web_pages_apps or frappe.get_installed_apps(sort=True)
-			apps.reverse()
+			apps = list(reversed(frappe.local.flags.web_pages_apps or frappe.get_installed_apps()))
```

Comment:
This is being done instead of `apps.reverse` because we do NOT want to mutate the original object.